cms/envs/common.py

```diff
@@ -208,6 +208,12 @@
 
     # Show Language selector
     'SHOW_LANGUAGE_SELECTOR': False,
+
+    # Temporary feature flag for disabling saving of subsection grades.
+    # There is also an advanced setting in the course module.  This
+    # feature flag is used only to toggle whether the advanced setting
+    # appears in Studio.
+    'ENABLE_SUBSECTION_GRADES_SAVED': False,
```

Comment:
In LMS, as long as we support multiple code paths using the course' advanced setting, it's not necessary to double-it-up with a feature flag as well, correct?  And, if we do go forward with a staged rollout of this work, we'll need to make sure that all conditional logic is behind the advanced setting - and not being a feature flag, right?